boardgame-ui/src/components/timer/PlayerTimer.vue

```diff
@@ -35,7 +35,7 @@ export default {
   },
   computed: {
     textColor() {
-      const match = this.player_timer.color.match(/^#([0-9a-fA-F]{2})([0-9a-fA-F]{2})([0-9a-fA-F]{2})$/);
+      const match = this.player_timer.color.match(/^#([0-9a-fA-F]{2})([0-9a-fA-F]{2})([0-9a-fA-F]{2})([0-9a-fA-F]{2})?$/);
```

Comment:
maybe we should use the transparency in some way if there is one ?

Reply:
why not but maybe in another pull request